dakara_server/library/tests_command_createworks.py

```diff
@@ -0,0 +1,147 @@
+import os
+
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from django.test import TestCase
+from .models import WorkType, Work
+
+APP_DIR = os.path.dirname(os.path.abspath(__file__))
+RESSOURCES_DIR = os.path.join('tests_ressources', 'work_files')
+DIR_WORK_FILES = os.path.join(APP_DIR, RESSOURCES_DIR)
+
+
+class CommandsTestCase(TestCase):
+    def setUp(self):
+        WorkType.objects.create(query_name="WorkType 1")
+
+    def test_createworks_from_correct_work_file(self):
+        """Test create works command from a correctly structured work file
+        """
```

Comment:
If you assert the final state of the database at the end of the test, you must assert its initial state at the beginning too. This proves that the command effectively did something. This is true for all your tests.